jwst/datamodels/schemas/dark.schema.yaml

```diff
@@ -13,8 +16,8 @@ allOf:
       title: 2-D data quality array for all planes
       fits_hdu: DQ
       default: 0
-      ndim: 2
-      datatype: uint32
+      ndim: 3
+      datatype: int16
```

Comment:
Both of these lines should revert to their original values of 2 and uint32, respectively.